src/Common/src/TypeSystem/Common/TypeSystemConstaintsHelpers.cs

```diff
@@ -47,6 +42,53 @@ private static bool VerifyGenericParamConstraint(Instantiation typeInstantiation
             return true;
         }
 
+        // Used to determine whether a type parameter used to instantiate another type parameter with a specific special 
+        // constraint satisfies that constraint.
+        private static bool CheckGenericSpecialConstraint(TypeDesc type, GenericConstraints specialConstraint)
```

Comment:
Could you please add coverage for this to the type system unit tests?

https://github.com/dotnet/corert/blob/master/src/ILCompiler.TypeSystem/tests/ConstraintsValidationTest.cs

Reply:
Yes, of course. Unfortunately, I will probably not be able to do so until Monday.

Reply:
Sorry for the delay, I finally came around to adding the tests today.